yarn.lock

```diff
@@ -12789,6 +12789,13 @@ ovh-api-services@^9.39.1:
   dependencies:
     lodash "^4.17.15"
 
+ovh-api-services@^9.39.0:
```

Comment:
Should be `9.39.1` as in the base branch

Reply:
will take care of it in base branch feat/veeam-backup before merging into develop